.github/workflows/docker-stable.yml

```
@@ -15,14 +15,22 @@ on:
 
   dockerize-stable:
     needs: [get-environment]
     runs-on: ubuntu-24.04
-    if: ${{ needs.get-environment.outputs.skip_workflow == 'false' && !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && needs.get-environment.outputs.stability == 'stable' && github.ref_name == 'master' }}
+    if: |
+      needs.get-environment.outputs.skip_workflow == 'false' &&
+      !cancelled() &&
+      !contains(needs.*.result, 'failure') &&
+      !contains(needs.*.result, 'cancelled') &&
+      needs.get-environment.outputs.stability == 'stable' &&
+      github.ref_name == 'master' &&
```

actionlint (1.7.4)

property "get-version" is not defined in object type {get-environment: {outputs: {is_cloud: string; is_nightly: string; is_targeting_feature_branch: string; latest_major_version: string; major_version: string; minor_version: string; os_and_database_matrix: string; pull_request_environment: string; release: string; release_type: string; skip_workflow: string; stability: string; target_stability: string}; result: string}}

(expression)